Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -135,4 +134,108 @@ class CustomerIOImplementation: CustomerIOInstance {
     ) {
         DataPipeline.shared.trackMetric(deliveryID: deliveryID, event: event, deviceToken: deviceToken)
     }
+
+    func handleQueueBacklog() {
+        let allStoredTasks = backgroundQueue.getAllStoredTasks()
+        if allStoredTasks.count <= 0 {
+            logger.info("No tasks pending in the background queue to be executed.")
```

Comment:
Probably could use more detail, like its CIO migration queue etc so we dont confuse what queue is the log about

Reply:
That makes sense.